### DISCUSSION TITLE
problem with the "signOut" using the "auth0" provider (with email + password connection)

### DISCUSSION THREAD

**Discussion by amaryassa** [nextauthjs/next-auth#1076] [Help]
provider : auth0
connection : with email and password.

after the first connection.  
we disconnect, then when we reconnect we are not redirected to the connection page (mail + password) but it connects directly with the old informations, as if the signOut() didn't work.

and in auth0 dashbord ==>  Logs ==> Search :
for signIn() ==> **Success Login**
but for signOut() ==> **Success Exchange**

normally if everything is good when  signOut(), we must have **"Success Logout"**

![signOutProblem](https://user-images.githubusercontent.com/29547546/104131092-83f33d00-5374-11eb-9bc7-67c664e1ed83.gif)

**Comment by g0ng0n-dev** [comment]
Hi... I'm having the same issue but with Keycloak. I found this issue regarding auth0 signout https://github.com/nextauthjs/next-auth/issues/638 maybe can help with your issue
